server/util/aria.test.js

```diff
@@ -79,3 +83,97 @@ describe('Verify test hashes are matching as expected', () => {
     });
 });
+
+describe('Verify test hashes are matching as expected - (v2 test format)', () => {
+    it('should not match a v2 test format version with a v1 test format version', () => {
+        // Based on https://github.com/w3c/aria-at/commit/836fb2a997f5b2844035b8c934f8fda9833cd5b2
+        const v1TestsHash = hashTests(commandButton_V20230823_V1);
+
+        // Based on https://github.com/w3c/aria-at/commit/d9a19f815d0f21194023b1c5919eb3b04d5c1ab7
+        const v2TestsHash_1 = hashTests(commandButton_V20231206_V2);
+        // Based on https://github.com/w3c/aria-at/commit/565a87b4111acebdb883d187b581e82c42a73844
+        const v2TestsHash_2 = hashTests(commandButton_V20231213_V2);
+
+        expect(v1TestsHash).not.toEqual(v2TestsHash_1);
+        expect(v1TestsHash).not.toEqual(v2TestsHash_2);
+    });
+
+    it('should not completely match a collection of tests hash with updated AT settings', () => {
+        // The difference between them is that there have been updated setting identifiers for VoiceOver tests;
+        // quickNavOn -> singleQuickKeyNavOn
+        // Compare with https://github.com/w3c/aria-at/compare/d9a19f8..565a87b
+
+        // Based on https://github.com/w3c/aria-at/commit/d9a19f815d0f21194023b1c5919eb3b04d5c1ab7
+        const v2TestsHash_1 = hashTests(commandButton_V20231206_V2);
+        // Based on https://github.com/w3c/aria-at/commit/565a87b4111acebdb883d187b581e82c42a73844
+        const v2TestsHash_2 = hashTests(commandButton_V20231213_V2);
+
+        expect(v2TestsHash_1).not.toEqual(v2TestsHash_2);
+    });
+
+    it('should match collection of tests not a part of updated AT settings', () => {
+        // Only tests[].at.key === 'voiceover_macos' was affected, not nvda or jaws
+        const filterFn = t => t.at.key !== 'voiceover_macos';
+
+        // Based on https://github.com/w3c/aria-at/commit/d9a19f815d0f21194023b1c5919eb3b04d5c1ab7
+        const v2TestsHash_1 = hashTests(
+            commandButton_V20231206_V2.filter(filterFn)
+        );
+
+        // Based on https://github.com/w3c/aria-at/commit/565a87b4111acebdb883d187b581e82c42a73844
+        const v2TestsHash_2 = hashTests(
+            commandButton_V20231213_V2.filter(filterFn)
+        );
+
+        expect(v2TestsHash_1).toEqual(v2TestsHash_2);
+    });
+
+    it('should match individual tests not a part of updated AT settings', () => {
+        // Pos 0 and pos 1 of both commandButtonV2 tests point to JAWS and NVDA tests respectively
+        const v2TestHash_1_JAWS = hashTest(commandButton_V20231206_V2[0]);
+        const v2TestHash_2_JAWS = hashTest(commandButton_V20231213_V2[0]);
+
+        const v2TestHash_1_NVDA = hashTest(commandButton_V20231206_V2[1]);
+        const v2TestHash_2_NVDA = hashTest(commandButton_V20231213_V2[1]);
+
+        expect(v2TestHash_1_JAWS).toEqual(v2TestHash_2_JAWS);
+        expect(v2TestHash_1_NVDA).toEqual(v2TestHash_2_NVDA);
+    });
+
+    /* eslint-disable */
```

Comment:
Would it be possible to explain in a comment why disabling linting is needed?

Reply:
I think I included that in error when pushing this PR. I think I was ignoring the warnings to make the code changes here easier to read for myself.

But there is a point to be made here, that the default `printWidth` wrapping value made that block of code a bit hard to read (to me, and this does apply elsewhere, especially in frontend components). There was a discussion sometime back on this, which should be had to see what we think on updating that value.

Reply:
Removed the `eslint-disable` in https://github.com/w3c/aria-at-app/pull/880/commits/eeb2fb8f1e11176d3c4e27a40c05d2e2f1ce1ed4

Reply:
Yeah I'd love to update the tabWidth and printWidth 🤠